hack/k8s/helm/nuclio/templates/_helpers.tpl

```diff
@@ -107,3 +107,16 @@ NOTE: make sure to not quote here, because an empty string is false, but a quote
 {{- printf "%s-%s" (include "nuclio.dashboardName" .) .Values.dashboard.opa.name | trunc 63 | trimSuffix "-" -}}
 {{- end -}}
 {{- end -}}
+
+
+{{- define "nuclio.externalIPAddresses" -}}
+{{- if len .Values.dashboard.externalIPAddresses -}}
+{{- .Values.dashboard.externalIPAddresses | join "," | quote -}}
+{{- else if .Values.global.externalHostAddress -}}
+{{- .Values.global.externalHostAddress  -}}
```

Comment:
Precedence should be the opposite, general first, values later
The general block is there to allow charts that are using this chart as sub-chart to override the default values, e.g. with the current code, if at some point someone will add a default value for this field in Nuclio chart values I won't be able to override it when deploying Nuclio as part of the mlrun-kit
I see that the other fields that are using the global are like this as well, probably worth changing them as well